global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "7.0.103"
```

Comment:
Please revert this one - dotnet format is failing because of this version of sdk

Reply:
Do you know why I'm getting this error?

Severity	Code	Description	Project	File	Line	Suppression State
Warning		Unable to locate the .NET SDK version '7.0.103' as specified by global.json, please check that the specified version is installed.	GitVersion.MsBuild.Tests		1

Reply:
I have updated Visual Studio and thought it's backwards compatible

Reply:
You just need to have both 7.0.103 and 7.0.200 sdks installed, I suppose VS has installed the 7.0.200, you can manually install the other one